### DISCUSSION TITLE
Is it possible to assign tags on backfill from Dagit?

### DISCUSSION THREAD

**Discussion by Beaglefoot** [dagster-io/dagster#11578] [Q&A]
I'm on version 1.1.8 and looking for a way to lower priority of backfill runs with `dagster/priority` tag. I know CLI allows to do it, but what about UI?

**Comment by sryza** [comment]
Hi @Beaglefoot - mind filing an issue with this request? I think we should be able to get to it fairly soon.

**Reply by Beaglefoot** [reply]
Thanks @sryza
Here it is #11585.
